src/components/tableview/TableTable.vue

```diff
@@ -318,6 +323,13 @@ export default {
       }
       return null
     },
+    editorType(dt) {
+      switch (dt) {
+        case 'text': return 'textarea'
```

Comment:
I locally have a branch for that featureset (select for bool, textarea for text and datepicker for date(time)).
But i was stuck on the textarea, does it works for you ? (screen/video/gif would be great)
For me the size was buggy.

Reply:
I didn't tested textarea as it was already implemented, just added support to boolean with select

Reply:
Oh okay, i check your changes a second time.
I thought the `editorType` function will return the type which will be used for `tabulator`.

Reply:
@UtechtDustin I fixed textarea to work properly